ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -34,7 +34,9 @@ presubmits:
     - master
     - dockerless-build-temporary
     rerun_command: "/test e2e-aws"
-    always_run: true
+    # The abomination below is equivilent to `^((?!Documentation).)*$`. Since
+    # Go doesn't support negative lookaheads, we are stuck with the following.
+    run_if_changed: "^([^D]|D(D|oD|ocD|ocuD|ocum(D|e(D|n(D|t(D|aD|atD|atiD|atioD)))))*([^Do]|o[^Dc]|oc[^Du]|ocu[^Dm]|ocum([^De]|e([^Dn]|n([^Dt]|t([^Da]|a[^Dt]|at[^Di]|ati[^Do]|atio[^Dn]))))))*(D(D|oD|ocD|ocuD|ocum(D|e(D|n(D|t(D|aD|atD|atiD|atioD)))))*(o|oc|ocu|ocum(e(n(t(a|at|ati|atio)?)?)?)?)?)?$"
```

Comment:
How often does your repository change top-level directories? Seems like you could have called those out in a much more simple manner.

Reply:
> How often does your repository change top-level directories?

We have a top-level change in flight now with openshift/release#120 ;).  But we'll need CI changes in other places to adapt to that anyway, so I wouldn't mind switching to a whitelist approach here too.  I'm also fine with the blacklist approach.